src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -1957,5 +1846,14 @@ internal DeviceOrientationChangedSignalType TaskDeviceOrientationChangedSignal()
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
+
+        private void DestroySignal(Signal signal, Delegate callbackDelegate)
```

Comment:
The repeated pattern includes `signal = null`.
How about adding `ref` keyword to make signal to null?
```C#
private void DestorySignal(ref Signal signal, Delegate callbackDelegate)
{
    if (signal != null)
    {
        if (callbackDelegate != null)
        {
            signal.Disconnect(callbackDelegate);
        }
        signal.Dispose();
        signal = null;
    }
}
```